go.mod

```diff
@@ -99,7 +99,10 @@ require (
 	github.com/golang/protobuf v1.5.4 // indirect
 	github.com/google/gnostic-models v0.6.9 // indirect
 	github.com/google/go-configfs-tsm v0.2.2 // indirect
+	github.com/google/go-sev-guest v0.9.3 // indirect
+	github.com/google/go-tdx-guest v0.3.1 // indirect
```

Comment:
question: its generally felt these dependencies are a requirement?

Reply:
Can we get rid of them? I think `go-tpm-tools` brings them in.

Reply:
is go-tpm-tools required?

Reply:
I guess we can try to trim this back down the road to just go-tpm?